app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+  def new
```

Comment:
Ah!  Good catch, surprised I didn't notice that when testing.  Weird it was letting me log in without problems.  For now I added "skip_before_action" to the controller until I refactor the code to not use that.